pkg/server/gitproviders/service.go

```diff
@@ -173,6 +173,8 @@ func (s *GitProviderService) newGitProvider(config *gitprovider.GitProviderConfi
 		return gitprovider.NewAzureDevOpsGitProvider(config.Token, baseApiUrl), nil
 	case "aws-codecommit":
 		return gitprovider.NewAwsCodeCommitGitProvider(baseApiUrl), nil
+	case "gogs":
+		return gitprovider.NewGogsGitProvider(config.Token, *config.BaseApiUrl), nil
```

Comment:
Use `baseApiUrl` instead of  `*config.BaseApiUrl`

Reply:
done